src/server/game/Groups/Group.cpp

```diff
@@ -2092,7 +2092,41 @@ void Group::ResetInstances(uint8 method, bool isRaid, Player* SendMsgTo)
         {
             // do not reset the instance, just unbind if others are permanently bound to it
             if (isEmpty && instanceSave->CanReset())
+            {
+                if (map && this->isRaidGroup() && map->IsDungeon() && SendMsgTo)
+                {
+                    AreaTrigger const * const instanceEntrance = sObjectMgr->GetGoBackTrigger(map->GetId());
+
+                    if (!instanceEntrance)
+                        TC_LOG_DEBUG("root", "Instance entrance not found for maps %u", map->GetId());
+                    else
+                    {
+                        WorldSafeLocsEntry const * graveyardLocation = sObjectMgr->GetClosestGraveyard(instanceEntrance->target_X, instanceEntrance->target_Y, instanceEntrance->target_Z, instanceEntrance->target_mapId, SendMsgTo->GetTeam());;
+                        uint32 const zoneId = sMapMgr->GetZoneId(graveyardLocation->map_id, graveyardLocation->x, graveyardLocation->y, graveyardLocation->z);
+
+                        for (const MemberSlot &member : this->GetMemberSlots())
+                        {
+                            if (!ObjectAccessor::FindConnectedPlayer(member.guid))
+                            {
+                                PreparedStatement *stmt = CharacterDatabase.GetPreparedStatement(CHAR_UPD_CHARACTER_POSITION_BY_MAPID);
+
+                                stmt->setFloat(0, graveyardLocation->x);
+                                stmt->setFloat(1, graveyardLocation->y);
+                                stmt->setFloat(2, graveyardLocation->z);
+                                stmt->setFloat(3, instanceEntrance->target_Orientation);
```

Comment:
is this some leftover or is it because graveyardLocation doesn't have any orientation ?

Reply:
Yes, that is because no orientation is given by graveyardLocation

Reply:
I checked how Player::RepopAtGraveyard() works and it keeps the current player orientation but I would say in this case it doesn't really matter